### PR TITLE
Remove support for WP8 and Win8.

### PR DESCRIPTION
**Pull request by jlaanstra**
Remove support for WP8 and Win8 for Splat 2.0.

### CODE REVIEW

File: Splat-VisualStudio.sln [hunk 1]
```diff
@@ -23,7 +21,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		Splat.nuspec = Splat.nuspec
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Splat-wpa81", "Splat\Splat-wpa81.csproj", "{E3C22E93-6CA9-4781-A3DE-D6DF775E8AE7}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Splat-win81+wpa81", "Splat\Splat-win81+wpa81.csproj", "{E3C22E93-6CA9-4781-A3DE-D6DF775E8AE7}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Splat-UAP", "Splat\Splat-UAP.csproj", "{583DDF7A-50DB-449E-9409-100DC8679136}"
```
Comment (paulcbetts) [inline Splat-VisualStudio.sln]: Is `UAP` really the profile name?

Comment (nigel-sampson) [inline Splat-VisualStudio.sln]: I didn't think the nuget team had published one yet

Comment (jlaanstra) [inline Splat-VisualStudio.sln]: Probably not. Lemme check for the correct name.

**Comment by flagbug** [comment]
We'll leave WP8 as long as Visual Studio officialy supports it, but I've removed Win80 support in #136

**Pull request closed without merging**